programs/odbc-bridge/ODBCConnectionFactory.h

```diff
@@ -50,6 +53,7 @@ namespace DB
 
 static constexpr inline auto ODBC_CONNECT_TIMEOUT = 100;
 static constexpr inline auto ODBC_POOL_WAIT_TIMEOUT = 10000;
+static constexpr auto ODBC_CHECK_CONNECTION_QUERY = "SELECT 1";
```

Comment:
Will not work on oracle: `ORA-00923: FROM keyword not found where expected `

Reply:
Have no idea how to solve it. Most of other databases will complain on Oracle's `SELECT 1 FROM dual`

Making it configurable sounds like overkill.